packages/beacon-node/src/api/impl/beacon/state/index.ts

```
@@ -130,6 +130,37 @@ export function getBeaconStateApi({
       return this.getStateValidators(args, context);
     },
 
+    async postStateValidatorIdentities({stateId, validatorIds = []}) {
+      const {state, executionOptimistic, finalized} = await getStateResponse(chain, stateId);
+      const {pubkey2index} = chain.getHeadState().epochCtx;
+
+      let validatorIdentities: routes.beacon.ValidatorIdentities;
+
+      if (validatorIds.length) {
+        validatorIdentities = [];
+        for (const id of validatorIds) {
+          const resp = getStateValidatorIndex(id, state, pubkey2index);
```

Do we want to add a log in case `resp.valid === false`?

---

Might be helpful to debug issues, right now we just silently ignore those